lib/helpers.coffee

```diff
@@ -48,7 +48,8 @@ module.exports = Helpers =
         return reject(error) if error and error.code is 'ENOENT'
         handle()
         if error.code is 'EACCES'
-          error = new Error("Failed to spawn command `#{command}`. Make sure it's a file, not a directory and it's executable.")
+          error = new Error("Failed to spawn command `#{command}`. Make sure " +
```

Comment:
It's also a very long line, do you want it split in another section? (after the period?)